cypress/support/step_definition/course/createCourse/createCourse.spec.js

```diff
@@ -1,10 +1,10 @@
 
-import Commom_Logins from '../../../page/common/logins'
+import Common_Logins from '../../../page/common/logins'
 import Create_Course from '../../../page/course/createCoursePageObject'
 import Room_Overview from '../../../page/common/goToRoomOverview'
 
 Given('I am logged in as a teacher', () => {
-    const loginAsTeacher = new Commom_Logins()
+    const loginAsTeacher = new Common_Logins()
```

Comment:
This should be outside the step. Because it's redundant.

Reply:
I agree, I did this for my tests (as a test :D) already and it's not needed to mentioned the login step again and again.

Reply:
Solved.